vaadin-checkbox-flow-demo/src/main/java/com/vaadin/flow/component/checkbox/demo/CheckboxView.java

```diff
@@ -32,12 +32,18 @@
 import com.vaadin.flow.demo.DemoView;
 import com.vaadin.flow.router.Route;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.LinkedHashSet;
 import java.util.List;
+import java.util.Objects;
 import java.util.Set;
 
+import static com.vaadin.flow.component.checkbox.demo.CheckboxView.DishData.getDishOfTheDay;
+import static com.vaadin.flow.component.checkbox.demo.CheckboxView.DishData.getDishes;
```

Comment:
Un-needed static imports now that DishData is an internal class.

Reply:
Static import is used here because I want to have `getDishes()` instead of `DishData.getDishes()` in the demo code snippet.

Reply:
As a user reading the demo code I would rather see `DishData.getDishes()` so I wouldn't hunt for getDishes() method

Reply:
Alright, will fix it